packages/cli/src/lib/commands/topic.ts

```diff
@@ -1,9 +1,18 @@
+/* eslint-disable no-console */
 import { CommandDefinition } from "../../types";
 import { isDevelopment } from "../../utils/envs";
 import { getHostClient, getReadStreamFromFile } from "../common";
 import { profileConfig } from "../config";
 import { displayEntity, displayStream } from "../output";
 
+const validateTopicName = (topicName: string) => {
+    if (topicName.match(/^[a-zA-Z0-9_-]*$/)) {
+        return;
+    }
+
+    throw new Error("Invalid topic name");
```

Comment:
Some characters are acceptable and some are not, for example, slash throws an error but backslash is ok:

![image](https://user-images.githubusercontent.com/61054769/183388773-42a9be15-9e8a-45fb-b298-7b7fc2c20f99.png)

Should we be more precise about the characters that cannot be used in the topic name? Maybe something like this:

```suggestion
    throw new Error("Invalid topic name! Please do not use special characters in the topic name, such as: / ! @ # % ^ & * ( ) + | < >");
```

Reply:
actually `si topic send topic\name READM.md` creates `topicname` topic due to escape character `\`

Reply:
I don't actually see anything wrong with `/` if properly escaped.

Reply:
I'd stay with unreserved and safe url chars, also due to user friendly looking urls.